cl/assets/templates/includes/pagination.html

```diff
@@ -1,61 +1,73 @@
 {% load humanize %}
-
-{% if results.has_previous or results.has_next %}
-<div class="well v-offset-above-3 hidden-print">
-  <div class="row">
-    <div class="col-xs-2 col-sm-3">
-      {% if results.has_previous %}
-        <div class="text-left">
-          {% if webhooks %}
-            <button id="webhooks-paginator"
-               hx-get="{% url 'webhook_events-list' format="html" %}?debug={{debug}}&page={{ results.previous_page_number }}&webhook__event_type={{ type_filter }}&event_status={{ status_filter }}"
-               hx-target="#webhook_table_body"
+{% load extras %}
+<div class="col-xs-12" >
+  {% if page_obj.has_other_pages %}
+    <div class="well v-offset-above-3 hidden-print">
+      <div class="row">
+	      <div class="col-xs-3">
+          {% if page_obj.has_previous %}
+            {% if not hide_first %}
+              <a href="?{{ get_string }}page=1"
+                 rel="first" class="hidden-xs btn btn-default" >
+                <i class="fa fa-caret-left no-underline" ></i>
+                <i class="fa fa-caret-left no-underline" ></i>
+			          <span class="">First</span>
+              </a>
+            {% endif %}
+            <a href="?{{ get_string }}page={{ page_obj.previous_page_number }}"
                rel="prev"
-               class="btn btn-default">
-              <i class="fa fa-caret-left no-underline"></i>
-              <span class="hidden-xs hidden-sm">Previous</span>
-              <span class="hidden-xs hidden-md hidden-lg">Prev.</span>
-            </button>
-          {% else %}
-            <a href="?{{ get_string }}page={{ results.previous_page_number }}"
-               rel="prev"
-               class="btn btn-default">
-              <i class="fa fa-caret-left no-underline"></i>
-              <span class="hidden-xs hidden-sm">Previous</span>
-              <span class="hidden-xs hidden-md hidden-lg">Prev.</span>
+               class="btn btn-default" >
+              <i class="fa fa-caret-left no-underline" ></i>
+			        <span class="hidden-xs">Prev.</span>
             </a>
+          {% else %}
+            {% if not hide_first %}
+              <span class="hidden-xs btn btn-default" >
+                <i class="fa fa-caret-left no-underline" ></i>
+                <i class="fa fa-caret-left no-underline" ></i>
+			          <span class="">First</span>
+              </span>
+            {% endif %}
+            <span class="btn btn-default" >
+              <i class="fa fa-caret-left no-underline" ></i>
+			        <span class="hidden-xs">Prev.</span>
+            </span>
```

Comment:
If there's not a prev or next, let's not have a button at all instead of having a fake button. That'll keep with current design.